releasetool/commands/tag/python_tool.py

```diff
@@ -62,29 +62,36 @@ def create_release(ctx: Context) -> None:
     )
 
 
-def publish_to_pypi(ctx: Context) -> None:
+def publish_via_kokoro(ctx: Context) -> None:
     kokoro_url = "https://fusion.corp.google.com/projectanalysis/current/KOKORO/"
-    project = f"prod:cloud-devrel/client-libraries/{ctx.package_name}/release"
-    project_url = parse.urljoin(kokoro_url, parse.quote_plus(project))
 
-    pyperclip.copy(ctx.release_tag)
-
-    click.secho(
-        f"> Trigger the Kokoro build with the commitish below to publish to PyPI. The commitish has been copied to the clipboard.",
-        fg="cyan",
+    ctx.kokoro_job_name = f"cloud-devrel/client-libraries/{ctx.package_name}/release"
+    ctx.fusion_url = parse.urljoin(
+        kokoro_url, parse.quote_plus(f"prod:{ctx.kokoro_job_name}")
     )
-    click.secho(f"Build:\t\t{click.style(project_url, underline=True)}")
+
+    if ctx.interactive:
+        pyperclip.copy(ctx.release_tag)
+
+        click.secho(
+            f"> Trigger the Kokoro build with the commitish below to publish to PyPI. The commitish has been copied to the clipboard.",
+            fg="cyan",
+        )
+
+    click.secho(f"Kokoro build URL:\t\t{click.style(ctx.fusion_url, underline=True)}")
     click.secho(f"Commitish:\t{click.style(ctx.release_tag, bold=True)}")
```

Comment:
Should the commitish be printed in both interactive and non-interactive modes?

Reply:
Yes. We'll include it in the autorelease log.